### PR TITLE
Stub file for Two Fer exercise

### PR DESCRIPTION
**Pull request by mluisamc**

### CODE REVIEW

File: exercises/two-fer/two-fer.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+//
+// This is only a SKELETON file for the 'Two fer' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export class TwoFer {
+  return() {
```
Comment (SleeplessByte) [inline exercises/two-fer/two-fer.js]: The [spec](https://github.com/mluisamc/javascript/blob/stub-files/exercises/two-fer/two-fer.spec.js) does not expect a `class`, nor is this valid syntax.

Comment (mluisamc) [reply exercises/two-fer/two-fer.js]: Ok, modified!

File: exercises/two-fer/two-fer.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+//
+// This is only a SKELETON file for the 'Two fer' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export const twofer = () => {
```
Comment (SleeplessByte) [inline exercises/two-fer/two-fer.js]:
```suggestion
export const twoFer = () => {
```

The casing is incorrect. 

@tejasbubane maybe we want to use `export function twoFer` instead? Given that this is the first exercise and all?? What do you think?

File: exercises/two-fer/two-fer.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+  throw new Error("Remove this statement and implement this function");
+};
```
Comment (SleeplessByte) [inline exercises/two-fer/two-fer.js]: Can you add a newline here?

File: exercises/two-fer/two-fer.js (hunk 1 above)
Comment (tejasbubane) [reply exercises/two-fer/two-fer.js]: I always wonder why `export const xyz` is valid but `export class xyz` is not 🤷‍♂

File: exercises/two-fer/two-fer.js (hunk 2 above)
Comment (tejasbubane) [reply exercises/two-fer/two-fer.js]: I prefer `twoFer` given the JS convention of camelCase names & exercise name being `two-fer`.

File: exercises/two-fer/two-fer.js (hunk 1 above)
Comment (SleeplessByte) [reply exercises/two-fer/two-fer.js]: It is valid, I was talking about the `return() {`

File: exercises/two-fer/two-fer.js (hunk 2 above)
Comment (SleeplessByte) [reply exercises/two-fer/two-fer.js]: I actually meant `export const twoFer` vs `export function twoFer`. What are your thoughts on that @tejasbubane ?

Comment (tejasbubane) [reply exercises/two-fer/two-fer.js]: I think `const` will be fine.

**Comment by SleeplessByte** [comment]
Thank you!!

**Comment by SleeplessByte** [comment]
@mluisamc if you have 5 more minutes, would you be so kind and see if you can help us out by responding to [this issue](https://github.com/exercism/javascript/issues/741)?

**Pull request merged**